examples/C++/HelloWorldExampleTCP/HelloWorldSubscriber.h

```diff
@@ -37,7 +37,7 @@ class HelloWorldSubscriber {
 	HelloWorldSubscriber();
 	virtual ~HelloWorldSubscriber();
 	//!Initialize the subscriber
-	bool init();
+	bool init(const std::string &wan_ip, unsigned short port);
```

Comment:
TABS. Take the chance to change the whole file

Reply:
Fixed